### PR TITLE
1441 immunization upd

### PR DESCRIPTION
**Pull request by RamilGaripov**
Ticket: #_[ticket-number]_1441

### Description

- Fixed the `Immunization` section mapping

### Testing

_[Regular PRs:]_

- Local
  - [x] Made sure the immunization records show up in the FHIR resources, that the output length increased, and that the MR ends up displaying those records

### Release Plan
- [ ] Merge this

### CODE REVIEW

File: packages/fhir-converter/src/templates/cda/Resources/Immunization.hbs [hunk 1]
```diff
@@ -29,13 +29,18 @@
                     "lotNumber":"{{this.manufacturedMaterial.lotNumberText._}}",
                 {{/if}}
             {{/each}}
+            
+            {{#with this.manufacturedProduct.manufacturerOrganization}}
+                {{!-- manufacturer should be a reference to an Organization --}}
+                "manufacturer": {
```
Comment (RamilGaripov) [inline packages/fhir-converter/src/templates/cda/Resources/Immunization.hbs:35]: Note that this should be a Reference to an Organization (manufacturer). I haven't been able to make that work.. But the MR generation with this approach allows us to display the manufacturer's name, at least

File: packages/fhir-converter/src/templates/cda/Sections/Immunization.hbs [hunk 2]
```diff
@@ -1,6 +1,6 @@
-{{#if (contains (toString (toJsonString msg)) '2.16.840.1.113883.10.20.22.2.2.1')}}
-    {{#with (getFirstCdaSectionsByTemplateId msg '2.16.840.1.113883.10.20.22.2.2.1')}}
-            {{#each (toArray 2_16_840_1_113883_10_20_22_2_2_1.entry) as |immEntry|}}
+{{#if (contains (toString (toJsonString msg)) '2.16.840.1.113883.10.20.22.2.2')}}
+    {{#with (getFirstCdaSectionsByTemplateId msg '2.16.840.1.113883.10.20.22.2.2')}}
+            {{#each (toArray this.[2_16_840_1_113883_10_20_22_2_2].entry) as |immEntry|}}
```
Comment (Goncharo) [inline packages/fhir-converter/src/templates/cda/Sections/Immunization.hbs:3]: @RamilGaripov now following - why was the OID changed? https://build.fhir.org/ig/HL7/cda-ccda-2.2/StructureDefinition-2.16.840.1.113883.10.20.22.2.2.1.html

**Review by Goncharo — CHANGES_REQUESTED** [review]

File: packages/fhir-converter/src/templates/cda/Sections/Immunization.hbs (hunk 2 above)
Comment (RamilGaripov) [reply packages/fhir-converter/src/templates/cda/Sections/Immunization.hbs]: Included both OIDs to be used for immunizations

**Review by Goncharo — APPROVED** [review]
:shipit:

**Pull request merged**